### DISCUSSION TITLE
Changing Font Family

### DISCUSSION THREAD

**Discussion by kmanadkat** [wangchucheng/hugo-eureka#50] [Q&A]
How do I change font family from Lora to something else? I tried following but it didnot work:
1. Removed Lora & added other google font : `themes/eureka/layouts/partials/head.html`
```html
<link rel="preload"
  href="https://fonts.googleapis.com/css2?family=Hind+Guntur:wght@400;600;700&family=Noto+Serif+SC:wght@400;600;700&display=swap"
  as="style" onload="this.onload=null;this.rel='stylesheet'">
```
2. Removed Lora & added other google font : `themes/eureka/assets/css/tailwind.config.js`
```js
     'serif': ['Hind Guntur', 'Noto Serif SC', 'serif'],
```

no luck, is there any other way to add/remove font family..?

**Comment by wangchucheng** [comment]
If you do these steps right, I think you can change font family. But you may need to rebuild the tailwind assets. Maybe this can solve the problem.

**Reply by qmdx00** [reply]
Can you add rebuild tailwind assets script to package.json? I'm not familiar with  tailwind and postCss.

**Reply by wangchucheng** [reply]
Please remind me what does rebuild tailwind assets script refer to? Do you mean how to modify the css style?

**Reply by qmdx00** [reply]
Yes, I change `themes/eureka/assets/css/tailwind.config.js` file line 17 [tailwind.config.js](https://github.com/wangchucheng/hugo-eureka/blob/0167aa7006eb77d91af0bde70bbe20b6c9715517/assets/css/tailwind.config.js#L17) below
```
'mono': ['monospace', 'Menlo'],
```
how could I make it effective ?

**Reply by georgebridgeman** [reply]
I'm also looking to change the font family and am stuck at the same point. I've got (I think) all the dev requirements in place but can't figure out the command to rebuild the assets.
Any help would be greatly appreciated!

**Reply by wangchucheng** [reply]
If you need to modify the style, you can refer to the following steps:

1. Comment `build.useResourceCacheWhen` in `config/_default/config.yaml`.
2. `cd themes/eureka`
3. Follow instructions in [PostCSS | Hugo](https://gohugo.io/hugo-pipes/postcss/) to install necessary packages Hugo needs.
4. Run `npm install`.

Then when you run `hugo` or `hugo server` in your project, the assets will be regenerated.

Please note that this will **seriously** affect the generation speed of the website. It is recommended that after regenerating the assets, uncomment the `build.useResourceCacheWhen` in `config.yaml` to use generated cache.

**Reply by qmdx00** [reply]
Thanks a lot, I solved it. :smile:
